### ISSUE TITLE
 Import declaration conflicts with local declaration of 'CloseEvent'.

### ISSUE THREAD

**Issue by ZhangZoo** [pladaria/reconnecting-websocket#118]
An error was reported in the project of create-react-app：“ Import declaration conflicts with local declaration of 'CloseEvent'.”  The error is reported in “/node_modules/reconnecting-websocket/dist/reconnecting-websocket.d.ts (7,10)” . ErrorEvent and Event  is also wrong. Sincerely hope to answer.

**Comment by alexjc** [comment]
This is a problem with Typescript 3.7 apparently.  It can be fixed manually with the following local changes to `reconnecting-websocket.d.ts`, but I'm not sure about the build process:

``` 
  7 import { CloseEvent as _CE, ErrorEvent as _EE, Event as _E, WebSocketEventListenerMap } from './events';
  8 export declare type Event = _E;
  9 export declare type ErrorEvent = _EE;
 10 export declare type CloseEvent = _CE;
```

**Comment by Suhail** [comment]
Yep, also having this issue when linting: `Import declaration conflicts with local declaration of 'CloseEvent'.ts(2440)`

**Comment by pladaria** [comment]
Fixed in #113
